### PR TITLE
techdebt: [SPG-107]: fix jdk11 unit tests

### PR DESCRIPTION
**Pull request by shivagowda**
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/34082)

Fixed
bazel test //400-rest:software.wings.service.impl.tests2
bazel test //400-rest:io.harness.delegate.resources.tests

Unable to reproduce:
bazel test //400-rest:software.wings.service.impl.tests5

### CODE REVIEW

**Review by pranil-harness — DISMISSED** [review]
> Reviewed 4 of 4 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34082)* status: :shipit: complete! ✓

File: 400-rest/src/test/java/software/wings/service/impl/EcsDelegateRegistrationTest.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import static io.harness.rule.OwnerRule.ADWAIT;
 import static io.harness.rule.OwnerRule.MARKO;
 
+import static org.mockito.Mockito.*;
```
Comment (hemanthmantri) [inline 400-rest/src/test/java/software/wings/service/impl/EcsDelegateRegistrationTest.java:14]: prefer importing only what's required.

Comment (shivagowda) [reply 400-rest/src/test/java/software/wings/service/impl/EcsDelegateRegistrationTest.java]: done

**Review by hemanthmantri — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34082#-:-N4-YQuj3AuD7tzoNibw:bnfp4nl)* status: 1 file(s) to review. 1. unresolved discussions.

**Review by ershadalim — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34082#-:-N40PGnO6PeEtE_V1Wlq:bnfp4nl)* status: 1. unresolved discussions.

**Pull request merged**